ldap/servers/plugins/replication/repl5_inc_protocol.c

```diff
@@ -325,8 +325,8 @@ repl5_inc_result_threadmain(void *param)
 
             conn_get_error_ex(conn, &operation_code, &connection_error, &ldap_error_string);
             slapi_log_err(SLAPI_LOG_REPL, repl_plugin_name,
-                          "repl5_inc_result_threadmain - Result %d, %d, %d, %d, %s\n",
-                          operation_code, connection_error, conres, message_id, ldap_error_string);
+                          "repl5_inc_result_threadmain - sid \"%s\" - Result %d, %d, %d, %d, %s\n",
```

Comment:
Nit pick here, but I can you change the format to be:

    sid=\"%s\"

It's cleaner, and easier to parse if needed.  Do this for all the logging calls.

Reply:
That is right I changed it.